editable-grid-parent/editable-grid/src/main/java/org/wicketstuff/egrid/column/EditableGridActionsPanel.java

```diff
@@ -100,9 +100,10 @@ protected void updateAjaxAttributes(AjaxRequestAttributes attributes)
 			@Override
 			public void onClick(AjaxRequestTarget target)
 			{
+				EditableDataTable eventTarget = rowItem.findParent(EditableDataTable.class);
 				send(getPage(), Broadcast.BREADTH, new GridOperationData<T>(OperationType.DELETE,
```

Comment:
IMO, broadcasting the event to the page is sub-optimal. Using the EditableDataTable should be enough.